corva/application.py

```diff
@@ -1,2 +1,64 @@
+from typing import Any, Callable, List, Optional
+
+from corva.models.stream import StreamContext, StreamEvent
+from corva.network.api import Api
+from corva.settings import CorvaSettings, CORVA_SETTINGS
```

Comment:
This is confusing. I'd like to rename CORVA_SETTINGS to SETTINGS

Reply:
Yes, renamed the file, class and global variable